.storybook/Slider.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withState } from 'recompose'
+import { Slider } from '../src'
+
+const DemoSlider = withState('value', 'onChange', [32, 64])(Slider)
```

Comment:
I used recompose to allow the sliders to be interactive in Storybook, but can remove this if it's not something we'd want to use in other stories